mibs/bintec/mib-address.mib

```diff
@@ -0,0 +1,412 @@
+-- This file is corresponding to Release 10.2.3.100 from 2018/07/18 00:00:00
+
+
+---------------------------------------------------------------------------
+-- $RCSfile: mib-address,v $
+-- $Revision: 1.4 $
+-- $Date: 2013-08-12 11:44:47 $
+-- Author: RFC 3291
+---------------------------------------------------------------------------
+
+INET-ADDRESS-MIB DEFINITIONS ::= BEGIN

```

Comment:
And this one, to `INET-ADDRESS-MIB`, and in fact, all other files :) just name them to whats before `DEFINITIONS`

Reply:
Hi! Thanks for reviewing :blush:  I renamed all files according to your suggestion.